src/feature/RansacFeatureSetMatcher.h

```diff
@@ -88,14 +88,29 @@ class RansacFeatureSetMatcher: public AbstractFeatureSetMatcher {
 	/** Sets the maximum distance (dimensionless) for two descriptors to be considered as a valid match */ 
 	inline void setDistanceThreshold(double distanceThreshold)
 	    {m_distanceThreshold = distanceThreshold;}
-    
+
+    /** Sets the maximum number of correspondences to be used */
+    inline void setMaxCorrespondences(unsigned short maxCorrespondences)
+        {m_maxCorrespondences = maxCorrespondences;}
+
+    /** Whether we do flann for search or not */
+    inline void useFlann(bool use_flann)
+        {m_flann = use_flann;}
+
+    /** Number of leaf checks to use in flann */
+    inline void setFlannChecks(double checks)
+        {m_flannChecks = checks;}
+
     protected:
 	double m_successProbability; /**< The probability of finding a correct match if exists. */
 	double m_inlierProbability; /**< The probability of a generic correspondence to be an inlier. */
 	double m_distanceThreshold; /**< The maximum distance (dimensionless) for two descriptors to be considered as a valid match. This threshold depends on the actual distance employed. */
 	double m_rigidityThreshold; /**< The maximum value (in meters) of difference between the relative distance of two interest points. This implements a rigidity check in the RANSAC hypothesis generation. */
 	bool m_adaptive; /**< The flag to set the adaptive strategy to compute the number of RANSAC iterations (EXPERIMENTAL!!!). */
 	bool m_scoreInliersOnly; /** This flag causes the returned score to involve only the found inliers. */
+    unsigned short m_maxCorrespondences; /**< The maximum number of correpondences to be used. */
```

Comment:
This package style mixes tabs and spaces for indentation. In h files it uses tabs and in cpp file it uses spaces. This makes it easy to mistakenly indent with spaces instead of tabs which breaks the indentation when displayed with a different tab length. We should consider totally getting rid of the tabs in this package.